src/travel-unchained-2025.html

```
@@ -0,0 +1,194 @@
+                {% image "src/img/unchained/unchained-08.png", "ALT", "col-span-6 md:col-span-3", "h-full rounded-xl object-cover shadow-lg" %}
+                {% image "src/img/unchained/unchained-06.png", "ALT", "col-span-6 md:col-span-3", "h-full rounded-xl object-cover shadow-lg" %}
+            </div>
+        </div>
+    </section>
+
+    <section class="bg-gray-900 text-white relative overflow-hidden py-16 md:py-32">
+        <div class="max-w-screen-xl px-8 mx-auto relative z-10">
+            <div class="lg:flex items-center">
+                <div class="lg:w-1/2 lg:pr-24">
+                    <img class="w-20 md:w-32 mb-6" src="/img/unchained-logo-2025.svg" alt="">
+                    <h2 id="contact" class="md:text-5xl text-2xl font-black text-white" id="team">
```

HTMLHint (1.5.0)

[error] Duplicate of attribute name [ id ] was found.

(attr-no-duplication)